group1/comm/static/comm_tool.js

```diff
@@ -60,6 +61,34 @@ function startVideoChat() {
     alert("Please create a team to be able to video chat")
   }
 }
+*/
+
+
+//Set the sentinel for turning on/off the chat window
+let chatcounter = 0; 
+
+//Function for starting and stopping the video chat
+function startVideoChat(){
+  if(chatcounter == 0){
+    //increment the counter to 'on'
+    chatcounter = 1;
+    //set the URL to the video chat HTML api, and show the div containing it
+    let URL = '/communication/videochat'
+    $('#videoArea').removeClass('hide');
+    $('#videoFrame').attr('src', URL);
+    //shrink the scroll area that houses the messages to allow the shown div to fit
+    $('.scroll-area').css("max-width","50%");
```

Comment:
Missing a space between the comma and "50%"

Reply:
resolved